### DISCUSSION TITLE
Ai-Proctoring question

### DISCUSSION THREAD

**Discussion by simcc-repository** [SafeExamBrowser/seb-server#139] [Q&A] [answered]
I'm using SEB Server integrated with Moodle. In the configuration template, I noticed this feature and tried to research it online, but there isn't much information available. Based on the description, it seems to utilize client device detection to send events to the SEB Server. However, when I enabled this option and tested it with an SEB 3.8 client, it neither requested camera permissions nor sent any events to the event log. How can I configure this feature to make it work?

**Comment by danschlet** [comment, marked as answer]
This was a kind of experimental feature, which we only implemented in SEB for iOS for proctoring using Jitsi Meet. There is an old [YouTube video](https://www.youtube.com/watch?v=6GEn_NWnZe8) demonstrating the feature. Unfortunately there wasn't enough interest for this feature from the SEB user community (especially after most pandemic measures were discontinued) to implement it also in the Windows and macOS clients, so it's not supported on these platforms and deprecated for iOS.

**Reply by simcc-repository** [reply]
Thanks for the swift reply!!got it!
